js_modules/dagster-ui/packages/ui-core/src/resources/ResourceRoot.tsx

```diff
@@ -498,6 +501,63 @@ const ResourceUses: React.FC<{
           </Table>
         </Box>
       )}
+      {[
+        {
+          name: 'Schedules',
+          objects: resourceDetails.schedulesUsing,
+          icon: <Icon name="schedule" color={Colors.Gray400} />,
+        },
+        {
+          name: 'Sensors',
+          objects: resourceDetails.sensorsUsing,
+          icon: <Icon name="sensors" color={Colors.Gray400} />,
+        },
+      ]
+        .filter(({objects}) => objects.length > 0)
+        .map(({name, objects, icon}) => (
+          <Box key={name}>
+            <SectionHeader>
+              <Subheading>{name}</Subheading>
+            </SectionHeader>
+            <Table>
+              <thead>
+                <tr>
+                  <th>Name</th>
+                </tr>
+              </thead>
+              <tbody>
+                {objects.map((itemName) => {
+                  return (
+                    <tr key={name + ':' + itemName}>
+                      <td>
+                        <Box
+                          flex={{
+                            direction: 'row',
+                            alignItems: 'center',
+                            display: 'inline-flex',
+                            gap: 8,
+                          }}
+                          style={{maxWidth: '100%'}}
+                        >
+                          {icon}
+
+                          <Link
+                            to={workspacePathFromAddress(
+                              repoAddress,
+                              `/${name.toLowerCase()}/${itemName}`,
```

Comment:
Is the `toLowerCase()` needed? Does it sometimes fail to resolve the path correctly if it's not forced to lowercase?

Reply:
it's not needed, but it does feel a little strange to point the user's browser at `.../Schedules/...` or `.../Sensors/...` - more of an aesthetic preference than anything